src/FsAutoComplete/LspServers/AdaptiveFSharpLspServer.fs

```diff
@@ -930,16 +1064,28 @@ type AdaptiveFSharpLspServer(workspaceLoader: IWorkspaceLoader, lspClient: FShar
     |> Result.ofOption (fun () -> $"No parse results for {filePath}")
 
   let forceGetTypeCheckResults filePath =
-    let tyResults = getTypeCheckResults (filePath)
+    let rec doIt tryAgain =
```

Comment:
If we don't get type check results for a file, assume we need to mark it out of date and try again.